source/iface/platform.rst

```diff
@@ -147,21 +175,19 @@ the type of information.
    * - Descriptor
      - Return type
      - Description
-   * - profile
-     - string_class
-     - OpenCL profile
    * - version
-     - string_class
-     - OpenCL software driver version
+     - std::string
+     - Returns a backend-defined platform version.
    * - name
-     - string_class
-     - Device name of the platform
+     - std::string
+     - Returns the name of the platform.
    * - vendor
-     - string_class
-     - Vendor name
+     - std::string
+     - Returns the name of the vendor providing the platform.
    * - extensions
-     - vector_class<string_class>
-     - Extension names supported by the platform
+     - std::vector<std::string>
+     - Deprecated, use sycl::device::get_info() with sycl::info::device::aspects instead.
```

Comment:
Maybe it's better to add code format for **sycl::device::get_info()** and **sycl::info::device::aspects**
Like ``` ``sycl::device::get_info()`` ``` and  ``` ``sycl::info::device::aspects`` ```

Reply:
Fixed with : https://github.com/imvucic/SYCL_Reference/pull/4/commits/71c306ca2d8fbb43bfa09f496478889852722b16